pkg/util/utils_supported.go

```diff
@@ -7,103 +7,19 @@ package util
 //  should work to take darwin from this
```

Comment:
This is fine for a test PR; when the current project is to consolidate, I think it would be better to migrate all callers to call the c/storage version directly, so that future maintainers are less tempted to introduce divergence again.